ui/lib/core/addon/templates/components/confirmation-modal.hbs

```diff
@@ -6,25 +6,25 @@
   @showCloseButton={{true}}
 >
   <section class="modal-card-body">
-    <div class="box is-shadowless is-fullwidth is-sideless">
 
-      {{yield}}
+    {{yield}}
 
-      <div class="modal-confirm-section">
-        <p class="has-text-weight-semibold is-size-6">
-          Confirm
-        </p>
-        <p class="is-help">Type <strong>{{confirmText}}</strong> to confirm {{toConfirmMsg}}</p>
-        {{input
-            type="text"
-            value=confirmationInput
-            name="confirmationInput"
-            class="input has-margin-top"
-            autocomplete="off"
-            spellcheck="false"
-            data-test-confirmation-modal-input=testSelector
-          }}
-        </div>
+    <div class="modal-confirm-section">
+      <p class="has-text-weight-semibold is-size-6">
+        Confirm
+      </p>
+      <p class="is-help">Type <strong>{{confirmText}}</strong> to confirm {{toConfirmMsg}}</p>
+      {{input
+        type="text"
+        value=confirmationInput
+        name="confirmationInput"
+        class="input has-margin-top"
+        autocomplete="off"
+        spellcheck="false"
+        enter=(if (eq confirmationInput confirmText) onConfirm)
```

Comment:
handling enter and escape on the confirmation modal used by the replicationAction disable and demote.